src/components/LyricView.tsx

```diff
@@ -9,10 +9,10 @@ interface LyricViewProps {
 /**
  * Lyric Viewer React Functional Component.
  */
-const LyricViewer: React.FC<LyricViewProps> = (props) => {
+const LyricView: React.FC<LyricViewProps> = (props) => {
   var data;
   try {
-    data = require(`../../resources/Songs_&_Hymns_Of_Life/metadata/${props.songNumber}.json`);
+    data = require(`../resources/Songs_&_Hymns_Of_Life/metadata/${props.songNumber}.json`);
   } catch {
     return <h1 className="center">No Song Found</h1>;
   }
```

Comment:
On line 28 of this file maybe we should actually just get rid of that `Author` footer thing. The songs list has all the authors on it already, so we can avoid showing inconsistencies in the author until we finish the QA scrub through the lyric files